creme/geolocation/constants.py

```diff
@@ -1,3 +1,9 @@
 # -*- coding: utf-8 -*-
 
 DEFAULT_SEPARATING_NEIGHBOURS = 10000  # in meters
+
+DEFAULT_OSM_NOMINATIM_URL = 'https://nominatim.openstreetmap.org/search'
+DEFAULT_OSM_TILEMAP_URL = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
+DEFAULT_OSM_TILEMAP_COPYRIGHT = (
+    '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+)
```

Comment:
1. I'd prefer to add this as settings in settings.py (in the geolocation section). It's an official app, so I think its better. And add some comments to explain these values, their constraints etc...
2.  Why not split the copyright into URL & Label, to avoid users to give HTML ?

Reply:
Something like `...COPYRIGHT = '&copy; {link} contributors` + `...COPYRIGHT_URL=...` ?